apps/mull-ui/src/app/pages/create-event/date-calendar/date-calendar.scss

```diff
@@ -1,6 +1,7 @@
 @import 'variables';
 
 .date-calendar-container {
+  width: 100%;
```

Comment:
I see you added the styling to each class in create.event.scss. However now this line you added is dead code since its applied twice in two places. Please remove it.

Reply:
This line is still here

Reply:
I removed the line in `create.event.scss` and kept it in `date-calendar.scss` it made more sense to style the class in `date-calendar.scss`.